services/hooks/src/api.js

```diff
@@ -29,6 +29,7 @@ builder.declare({
   method: 'get',
   route: '/hooks',
   name: 'listHookGroups',
+  scopes: 'hooks:list-hook-groups',
```

Comment:
Maybe just `hooks:list-hooks:` (following whatever pattern is selected for `index:list-namespaces..`)